src/framework/parsers/glb-parser.js

```diff
@@ -1181,36 +1156,7 @@ const extensionSheen = function (data, material, textures) {
         extractTextureTransform(data.sheenRoughnessTexture, material, ['sheenGlossiness']);
```

Comment:
Shouldn't this instance of sheenGlossiness also be renamed to sheenGloss?

Reply:
Yes you're right. I've already done this in a followup PR which includes this and other changes.